From bb1dd6ace9f6112532e5c4ad7158f0703b5baf9a Mon Sep 17 00:00:00 2001
From: Lars Francke <git@lars-francke.de>
Date: Mon, 17 Feb 2025 16:42:49 +0100
Subject: Include jackson-dataformat-xml dependency.

This allows us to use XmlLayout for Log4jV2.
By including it here as a dependency we can make sure that we always have
the matching version and we don't need to include it manually later in the
build.
---
 server/pom.xml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/server/pom.xml b/server/pom.xml
index 8f1bb2bd72..4c443f9ac4 100644
--- a/server/pom.xml
+++ b/server/pom.xml
@@ -205,6 +205,11 @@
             <groupId>org.apache.logging.log4j</groupId>
             <artifactId>log4j-core</artifactId>
         </dependency>
+        <dependency>
+          <!-- This is an optional dependency of log4j which is needed to use XmlLayout -->
+          <groupId>com.fasterxml.jackson.dataformat</groupId>
+          <artifactId>jackson-dataformat-xml</artifactId>
+        </dependency>
         <dependency>
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-joda</artifactId>
